### PR TITLE
Add "Getting Started" section.

### PR DESCRIPTION
**Pull request by msporny**
This PR adds a "Getting Started" section that highlights usage of the new default `@vocab` and guides developers on how to use it and then upgrade to defining a vocabulary and JSON-LD Context file.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1111.html" title="Last updated on May 4, 2023, 4:40 PM UTC (78deec7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1111/c964156...78deec7.html" title="Last updated on May 4, 2023, 4:40 PM UTC (78deec7)">Diff</a>

### CODE REVIEW

File: index.html
Comment (TallTed) [inline index.html:1061]:
```suggestion
<a>verifiable credentials</a>. Developers can copy the
template below and paste it into common <a>verifiable credential</a>
tooling to start issuing, holding, and verifying prototype credentials.
```

File: index.html
Comment (TallTed) [inline index.html:1088]:
```suggestion
all of the credential properties are stable, it is advised that they generate
vocabulary and context files for their application and publish them at stable URLs
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Suggested changes look bigger than they are. Moved a paragraph to better logical placement. Tweaked some language.

File: index.html
Comment (andresuribe87) [inline index.html:1088]: Is it possible to link to resources that teach developers how to generate the vocabulary and context file?

**Review by andresuribe87 — COMMENTED** [review]
Much needed, thanks Manu

**Review by decentralgabe — APPROVED** [review]

File: index.html
Comment (dlongley) [inline index.html:1092]:
```suggestion
<a href="#extensibility"></a>. Alternatively, developers can reuse existing vocabulary
and context files that happen to fit their use case. They can explore the Verifiable
Credential Specifications Directory [[VC-SPECS]] for reusable resources.
        </p>
```

**Review by dlongley — APPROVED** [review]
Thanks! I added a couple sentences on reusing existing work and where it can be found.

**Review by OR13 — APPROVED** [review]

**Review by OR13 — CHANGES_REQUESTED** [review]
@msporny thanks for this PR, I closed the one that removes the broken links, please correct that issue in this PR as well.

**Review by brentzundel — APPROVED** [review]
approved with Ted and Dave's suggestions.

File: index.html
Comment (msporny) [reply index.html:1088]: Do you have a link to those resources? Unless they're exceedingly stable, we try to avoid linking to external content. For example, there is a video series I did that explains how to do this on YouTube as well as a few tutorials online to do so... however, those are on sites that may or may not stick around over the many years to decades that this specification might exist on the web (leading to broken links).

I guess we could link to the JSON-LD Specification? Would that work for you?

File: index.html
Comment (msporny) [inline index.html:1093]:
```suggestion
Credential Specifications Directory [[VC-SPECS]] for reusable vocabularies and context files.
```

File: index.html
Comment (andresuribe87) [reply index.html]: Yes, linking to the json-ld spec would make sense to me.

I would love for specs to include explanatory videos in order to make them more accessible and easier to understand for folks, but that's outside of scope.

**Review by TallTed — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Comment by msporny** [comment]
This PR is blocked based on a review of PR #1110 by @OR13. If @OR13 is ok w/ PR #1110, then this PR will be merged after that one is merged.

**Review by mprorock — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-05-10-vcwg#section1-2) on 2023-05-10

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.2.  update json schema to accurately reflect context defn (pr vc-data-model#1064)

_See github pull request [vc-data-model#1064](https://github.com/w3c/vc-data-model/pull/1064)._




**Manu Sporny:** continuing with other PRs: next item "update json schema to accurately reflect context definition".  

**Gabe Cohen:** this is blocked by a PR that's now been merged, which fixed the example context, which should reserve opposition.  

> *Michael Prorock:* +1.

**Orie Steele:** Manu opened a PR an example context that makes it no longer return a 404.  

**Brent Zundel:** The PR has been merged that makes it a 404.  

> *Michael Prorock:* i have approved.

**Orie Steele:** changing to no objection.  

_See github pull request [vc-data-model#1111](https://github.com/w3c/vc-data-model/pull/1111)._




**Gabe Cohen:** can this be merged with 1111?  

> *Michael Prorock:* please!!!!

**Orie Steele:** can be merged this.  

**Michael Prorock:** yes to merge this and with 1111.  

**Brent Zundel:** 1064 should be merged? No opposition. Merge it.  

**Gabe Cohen:** it's merged.  

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-05-10-vcwg#section1-3) on 2023-05-10

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.3.  Add "Getting Started" section (pr vc-data-model#1111)

_See github pull request [vc-data-model#1111](https://github.com/w3c/vc-data-model/pull/1111)._




**Manu Sporny:** next is 1111 and the one that Orie wish it to merged.  

**Brent Zundel:** Objections? None - merged.  

**Manu Sporny:** merges 1111.  

> *Gabe Cohen:* manu: is rebase and merge our practice? I have been squashing and merging.

</details>

**Pull request merged**
